pkg/lang/frontend/starlark/v1/testdata/test.envd

```diff
@@ -0,0 +1,8 @@
+def build():
+    base(os="ubuntu20.04", language="python3")
```

Comment:
This is a v0 test file.

I think we don't really run this test.

Reply:
https://github.com/tensorchord/envd/blob/main/pkg/lang/frontend/starlark/testdata/test.envd 

The code is also v0 in upstream/main.

But it does not matter because the test just gets the hash

```go
var _ = Describe("Starlark", func() {
	It("should be able to compile envd", func() {
		filename := "testdata/test.envd"
		hash, err := GetEnvdProgramHash(filename)
		Expect(err).NotTo(HaveOccurred())
		Expect(hash).To(Equal("cff1c81818116d42"))
	})
})
```

Reply:
I see. But I think we should change it to a v1 file. In case we don't support v0 in the future, then the interpreting will fail.

Reply:
I already updated it.